code/drasil-example/Drasil/DblPendulum/Concepts.hs

```diff
@@ -1,10 +1,11 @@
 module Drasil.DblPendulum.Concepts where
 
 import Language.Drasil
+import Data.Drasil.Theories.Physics (newtonSLRRC)
 
 
 concepts :: [IdeaDict]
-concepts = map nw [rod, horizontal, vertical] ++ map nw defs 
+concepts = map nw [rod, horizontal, vertical] ++ map nw defs ++ [nw newtonSLRRC] 
```

Comment:
When adding a single word at the end, use `: nw newtonSLRRC` instead. That's much more efficient.